docs/index.md

```diff
@@ -7,17 +7,35 @@ description: |-
 
 # Civo Provider
 
-The Civo provider is used to interact with the resources supported by Civo. The provider needs to be configured with the proper credentials before it can be used.
+The Civo provider is used to interact with the resources supported by Civo. 
 
 Use the navigation to the left to read about the available resources.
 
+## Configuration
+
+The provider will use the credentials of the [Civo CLI](https://github.com/civo/cli) (stored in ` ~/.civo.json`) if no other credentials have been set up. The credentials preference is the following:
+
+1. Environment variable (`CIVO_TOKEN`).
+1. Token provided via a credentials file (See `credentials_file` input [below](#credentials_file))
+1. [CLI](https://github.com/civo/cli) configuration (`~/.civo.json`)
+
+### Obtaining a token
+
+First you will need to create a [Civo Account](https://dashboard.civo.com/signup) and then you can do the following:
+
+* If you can want to configure your credentials with the CLI, instructions are [here](https://www.civo.com/docs/overview/civo-cli#add-an-api-key-to-civo-cli)
+* To fetch an API key go to the [security section](https://dashboard.civo.com/security) on the dashboard
+
+
+
 ## Example Usage
 
+### Simplest usage
+
+In this example the provider will look for credentials set by the CLI and use LON1 region as default, or use the environment variable `CIVO_TOKEN` if set.
+
```

Comment:
we can also add something like this for more spoonfeeding to user

```
You can export the env variable by running

export CIVO_TOKEN=your_token
```
this would be more helpful

Reply:
Great point, I was assuming most people would get that, but it doesn't hurt to be abundantly explicit.